data/data/ovirt/affinity_group/main.tf

```diff
@@ -0,0 +1,18 @@
+resource "ovirt_affinity_group" "cluster_masters_affinity_group" {
+  count        = length(var.ovirt_control_plane_affinity_groups)
+  name         = var.ovirt_control_plane_affinity_groups[count.index]["name"]
+  cluster_id   = var.ovirt_cluster_id
+  priority     = var.ovirt_control_plane_affinity_groups[count.index]["priority"]
+  vm_positive  = false
+  vm_enforcing = false
+  vm_list      = var.masters_ids
+}
+
+resource "ovirt_affinity_group" "cluster_affinity_group" {
+  count        = length(var.ovirt_compute_affinity_groups)
+  name         = var.ovirt_compute_affinity_groups[count.index]["name"]
+  cluster_id   = var.ovirt_cluster_id
+  priority     = var.ovirt_compute_affinity_groups[count.index]["priority"]
+  vm_positive  = false
+  vm_enforcing = false
```

Comment:
This option should be customizable. Furthermore, there should be a customizable description with the following text:

> AG for spreading the load out to multiple hosts without hard enforcement.

Reply:
+1